solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -2669,6 +2669,18 @@ public static CloudSolrClient getCloudSolrClient(String zkHost) {
         .build();
   }
 
+  /**
+   * This method <i>may</i> randomize unspecified aspects of the resulting SolrClient. Tests that do
+   * not wish to have any randomized behavior should use the {@link
+   * org.apache.solr.client.solrj.impl.CloudSolrClient.Builder} class directly
+   */
+  public static CloudSolrClient getCloudSolrClient(String zkHost, String defaultCollection) {
```

Comment:
I honestly don't like these methods.  In a sense it hides a builder, -- it's as if we don't actually like builders and instead we like many overloaded methods that takes all sorts of parameters that are hard to decipher at the call site.  Yuck!  We should should want to embrace builders!

Reply:
I could get behind eliminating these....    It makes SolrTestCaseJ4 super long and busy.....

Reply:
I opened up SOLR-16604 as a seperate JIRA to track this.

Reply:
My ask in this PR is only not to introduce new ones.

Reply:
Makes sense..   I went and looked, and they all pretty much were only used by 1 test method...